app/views/helpers/index/normal/entry_header.phtml

```diff
@@ -44,6 +44,6 @@
 			?></span><?php
 		endif;
 	?></a></li>
-	<?php if ($topline_date) { ?><li class="item date"><?= $this->entry->date() ?> </li><?php } ?>
```

Comment:
Minor: We used to have a space after the date, to avoid being too close to the border, especially when the external link icon is not displayed:
![image](https://user-images.githubusercontent.com/1008324/87403314-00bb7f00-c5bd-11ea-9331-2e66f5f18b15.png)

Reply:
I've fixed that but I feel that this should be handled by CSS instead of relying on spacing. My 2¢

Reply:
Indeed, and I cannot remember why it was like that, but let's take that separately :-)